shared/src/userData.ts

```diff
@@ -46,7 +47,7 @@ export const createEmptyBusiness = (id?: string): Business => {
       roadmapOpenSteps: [],
       hiddenCertificationIds: [],
       hiddenFundingIds: [],
-      visibleSidebarCards: [],
+      visibleSidebarCards: [SIDEBAR_CARDS.notRegistered, SIDEBAR_CARDS.notRegisteredExistingAccount],
```

Comment:
[pebble] I see that the logic in `dashboard.tsx` is overwriting `visibleSidebarCards` to whichever sidebar card needs to be present based on persona and operatingPhase. What is the value in empty userData containing these values when they are being overwritten? Additionally, by including both of these values, we are for a brief second creating a state within userData that should be impossible. A user shouldn't have values that are for both starting/foreign and existing personas. Let me know if this makes sense or if I'm misunderstanding the intention of it.

Reply:
I think would also change the setup for all the tests, because why would/should we have a user that is `starting` have the `not-registered-existing-account` in their userData? That should be an impossible state because `not-registered-existing-account` vs `not-registered` are based on persona and only one of those should exist?

Reply:
I also am looking at the changes and I see that the names `not-registered-existing-account` and `not-registered` were preexisting but I think we should refactor the name `not-registered-existing-account` to something that indicates that it's for owning businesses because at first glance it looks like we already know they have account with OIT somehow and are nudging them based on that information from that name. This should be a follow-up but just a thought.

Reply:
Removed